backend/user/models.py

```diff
@@ -24,6 +25,7 @@ class Bidder(MainModel):
     bidder_number = models.IntegerField(unique=True, blank=True, null=False)
     is_verified = models.BooleanField(default=False)
     is_blacklisted = models.BooleanField(default=False)
+    saved_list = models.ManyToManyField(Vehicle)
```

Comment:
@kevinrczhang One last thing, there's already a `SavedUnits` model in the `vehicles` models